README.md

```diff
@@ -25,55 +26,27 @@ The Busola also includes React and Angular libraries:
 - [`npm`](https://www.npmjs.com/): >= 6.4.0
 - [`node`](https://nodejs.org/en/): >= 12.0.0
 
-## Installation
-
-1. Install [Kyma](https://kyma-project.io/docs/master/root/kyma/#installation-install-kyma-locally) as a backing service for your local instance of Busola. Make sure you import certificates into your operating system and mark them as trusted. Otherwise, you cannot access the applications hosted in the `kyma.local` domain.
-
-2. Install Busola dependencies. To install dependencies for the root and all UI projects, and prepare symlinks for local libraries within this repository, run the following command:
+## Usage with NPX
 
-   ```bash
-   npm run bootstrap
-   ```
+This mode has no hot-reload feature so it's not designed for the development. Hovewer, it gives you the ability to run the Busola in a few seconds without installing any dependencies.
```

Comment:
You do have to install dependencies, at least for the first run. I'd say you can run Busola without downloading code, building it etc.

Reply:
Are you sure? I just did it in a totally random folder outside the Busola repo...
![image](https://user-images.githubusercontent.com/22825433/112461433-3295ff80-8d60-11eb-8074-411d55fcc570.png)

Reply:
It's probably because it uses npm/npx cache, you have installed it before.

Reply:
Confirmed, my Windows machine:
![image](https://user-images.githubusercontent.com/10504661/112461934-c4057180-8d60-11eb-9ed8-804ebfef0a20.png)